src/config.ts

```diff
@@ -1,5 +1,7 @@
 const vars = {
     revalidatePeriod: 1,
+    pxPerRem: 18,
```

Comment:
Vi må huske på å redusere denne til 16 hvis vi går for å fjerne root front size-overriden vår. DS kjører 16px for root font-size og setter heller 18px på font-size i selve body. Vi har et møte i morgen for å avklare hva vi gjør her.